src/components/forums/forum/ForumContent.tsx

```diff
@@ -87,18 +89,17 @@ export function ForumContent(props: ForumContentProps) {
 
   const addAccessToken = async () => {
     setAddingAccessToken(true);
-    try {
-      const restriction = pubkeysToRestriction(
-        newForumAccessToken,
-        isSuccess(forumData.restriction) ? forumData.restriction : undefined
-      );
-      const currentIds = restrictionListToString(restriction);
-
-      const tx = await forumObject.setForumPostRestriction(
-        forumData.collectionId,
-        restriction
-      );
+    const restriction = pubkeysToRestriction(
```

Comment:
this `pubkeysToRestriction` and `restrictionListToString` functions don't have try catch blocks, I think it will cause a crash here if there's an error.  Should probably add in the try catch in the helper.

Reply:
I made some changes to handle possible errors in `pubkeysToRestriction`. I don't think `restrictionListToString` should be an issue as it doesn't seem to have any crashable segments. Let me know what you think